### PR TITLE
drop unused lapack dependency

### PR DESCRIPTION
**Pull request by jschleicher**
The dependency is not needed by our solver so I propose to drop it.

### CODE REVIEW

**Review by ct2034 — CHANGES_REQUESTED** [review]
Can you document this somewhere, so that we know, what manual steps to take whenever we generate the code again?

**Comment by jschleicher** [comment]
@ct2034 Added a quilt series to auto-apply the patches after the auto-update using your script.
No additional manual steps are necessary any more.

File: prbt_ikfast_manipulator_plugin/update_ikfast_plugin.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@ planning_group_name=manipulator
 ikfast_plugin_pkg=prbt_ikfast_manipulator_plugin
 base_link_name=prbt_base_link
 eef_link_name=prbt_flange
-ikfast_output_path=/home/ch/ros/pilz_ws/src/pilz_robots/prbt_ikfast_manipulator_plugin/src/prbt_manipulator_ikfast_solver.cpp
```
Comment (ct2034) [inline prbt_ikfast_manipulator_plugin/update_ikfast_plugin.sh:10]: This is also written by the script .. https://github.com/ros-planning/moveit/blob/master/moveit_kinematics/ikfast_kinematics_plugin/scripts/create_ikfast_moveit_plugin.py#L316
Maybe we could change that....

**Review by ct2034 — APPROVED** [review]
Cool! :+1:

File: prbt_ikfast_manipulator_plugin/update_ikfast_plugin.sh (hunk 1 above)
Comment (jschleicher) [reply prbt_ikfast_manipulator_plugin/update_ikfast_plugin.sh:10]: Indeed. Feel free to file a PR upstream.

**Comment by jschleicher** [comment]
@ct2034 ping
Could this be merged now?

**Pull request merged**